### DISCUSSION TITLE
Making narrow glyphs

### DISCUSSION THREAD

**Discussion by guichazan** [Chlumsky/msdf-atlas-gen#42] [General]
Hi!

Is it possible to make narrower glyphs to reduce blur when scaling at different sizes?

Best regards

   guich

**Comment by Chlumsky** [comment]
I don't know what you mean. Can you explain?

**Comment by guichazan** [comment]
Yes. I create a font at size 32. When i resample, it is drawn with blur. I would like to reduce the blur.
I played a bit with the options of the atlas generator but it does not make any noticeable difference.

**Comment by Chlumsky** [comment]
I still don't understand. What kind of font are you creating? SDF? Raster font? What are you resampling and how? Can you provide screenshots of the blur you're talking about?
